org.eclipse.lsp4e/schema/languageServer.exsd

```diff
@@ -22,6 +22,7 @@
          <sequence minOccurs="1" maxOccurs="unbounded">
             <element ref="server"/>
             <element ref="contentTypeMapping"/>
+            <element ref="markers" minOccurs="0" maxOccurs="1"/>
```

Comment:
I would rather see it as a child element under the `server` node.

Reply:
done